frontend/src/cards/MapCard.tsx

```diff
@@ -138,86 +174,75 @@ function MapCardWithKey(props: MapCardProps) {
-                        onClose={popover.close}
+                      <Button
+                        onClick={() => setSmallMultiplesDialogOpen(true)}
+                        color="primary"
                       >
-                        {["age", "all"].includes(props.currentBreakdown) && (
-                          <MenuItem disabled={true}>Age [unavailable]</MenuItem>
-                        )}
-                        {["sex", "all"].includes(props.currentBreakdown) && (
-                          <MenuItem disabled={true}>Sex [unavailable]</MenuItem>
-                        )}
-                        {["race_and_ethnicity", "all"].includes(
-                          props.currentBreakdown
-                        ) && (
-                          <>
-                            <MenuItem disabled={true}>Races</MenuItem>
-                            {breakdownValues.map((option) => (
-                              <MenuItem
-                                key={option}
-                                onClick={(e) => {
-                                  popover.close();
-                                  setBreakdownFilter(option);
-                                }}
-                              >
-                                {option}
-                              </MenuItem>
-                            ))}
-                          </>
-                        )}
-                      </Menu>
+                        Show full breakdown by{" "}
+                        {BREAKDOWN_VAR_DISPLAY_NAMES[activeBreakdown]}
+                      </Button>
                     </Grid>
                   </Grid>
                 </CardContent>
               </>
             )}
             <Divider />
-            <CardContent>
-              {queryResponse.dataIsMissing() && (
+            {queryResponse.dataIsMissing() && (
+              <CardContent>
                 <Alert severity="error">No data available</Alert>
+              </CardContent>
+            )}
+            {!queryResponse.dataIsMissing() &&
+              dataForActiveBreakdownFilter.length === 0 && (
+                <CardContent>
+                  <Alert severity="warning">
+                    No data available for filter: <b>{activeBreakdownFilter}</b>
+                  </Alert>
+                </CardContent>
               )}
-              {!queryResponse.dataIsMissing() && filteredData.length === 0 && (
-                <Alert severity="warning">
-                  No data available for filter: <b>{breakdownFilter}</b>
-                </Alert>
+            {!queryResponse.dataIsMissing() &&
+              dataForActiveBreakdownFilter.length !== 0 &&
+              props.metricConfig && (
+                <CardContent>
+                  <Alert severity="info">
+                    Note that legend changes between races. To see races with
+                    common legend, use show all breakdowns button.
```

Comment:
Can you make the text on alerts left-aligned? It looks a bit odd center-aligned when the second text line is very short

Reply:
I'm not even sure if we want this to be the final mock or if we just did this for UX testing... I have a bug already for these alerts so I'll add a note to address this too https://github.com/SatcherInstitute/health-equity-tracker/issues/431